pkg/discovery/network/network.go

```diff
@@ -19,14 +19,23 @@ package network
 import (
 	"fmt"
 
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
+
+	submarinerclientset "github.com/submariner-io/submariner-operator/pkg/client/clientset/versioned"
+)
+
+const (
+	OperatorNamespace = "submariner-operator"
```

Comment:
Please pass down the namespace to Discover as a parameter from the command, and use the existing constant in the pkg module: 
https://github.com/submariner-io/submariner-operator/blob/master/pkg/subctl/cmd/join.go#L225


The reason for this is that, eventually we may need to pass this down as a parameter, and avoiding the replication of the constant.

Reply:
and what about the submarinerName? keep it as const in the network.go file or pass as a parameter as well?